### PR TITLE
Add missing blocks for 1.17

### PR DESCRIPTION
**Pull request by yanlyub**
Is this the correct way to move forward?

The `ColorMapTest.testBlockNames` now finishes successfully. What are the next steps? Is it necessary to add different block states for the new blocks?

### CODE REVIEW

File: BlockMap-internal/src/main/resources/block-color-instructions-1_17.json [hunk 1]
```diff
@@ -573,7 +573,153 @@
...
+
+			"calcite": "calcite.png",
+			"cave_vines": "cave_vines.png",
+			"cave_vines_plant": "cave_vines_plant.png",
+			"copper_ore": "copper_ore.png",
+			"dirt_path": "dirt.png",
+			"dripstone_block": "dripstone_block.png",
+			"glow_lichen": "glow_lichen.png",
+			"hanging_roots": "hanging_roots.png",
+			"light":  "white_candle.png",
```
Comment (yanlyub) [inline BlockMap-internal/src/main/resources/block-color-instructions-1_17.json:711]: Could not find a texture for the Light Block

Comment (piegamesde) [reply BlockMap-internal/src/main/resources/block-color-instructions-1_17.json:711]: You can use `["fixed", "0xFFA26B23"]` to give it the color you want, or `["transparent"]` if it shouldn't be part of the map.

**Review by piegamesde — CHANGES_REQUESTED** [review]
Many thanks, this is indeed the correct path to move forward. Generally, you can save a good chunk of work by using the placeholders (defined at the top of the file), that will automatically duplicate for all possible combinations. (But you don't need to.)

The next step is to add the block states to make more tests work (I think the most interesting one should be `testDetailed`). Most of the time you simply need to add a `property=*` wildcard since most states don't affect the appearance of a block. Common exceptions to this are for example rotatable blocks.

**Comment by piegamesde** [comment]
Out of curiosity: can you please paste your log of `./gradlew build`? I'm getting unexpected errors and I'd like to know if they're reproducible.

**Comment by piegamesde** [comment]
So I found the root cause for the 1_16 errors: there was an implicit bump from 1.16.1 to 1.16.5 recently and the color map didn't get updated. Also thank you for telling me about the broken debug world, that explains those respective tests failing. I'll have a look.

**Comment by piegamesde** [comment]
Integrated into #67.

**Pull request closed without merging**